modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
+
+[id="machine-user-infra-machines-pxe_{context}"]
+= Creating more {op-system-first} machines by PXE or iPXE booting
+
+You can create more compute machines for your bare metal cluster by using
+PXE or iPXE booting.
+
+.Prerequisites
+
+* Obtain the URL of the Ignition config file for the compute machines for your
+cluster. You uploaded this file to your HTTP server during installation.
+* Obtain the URLs of the {op-system} ISO image, compressed metal BIOS, `kernel`,
+and `initramfs` files that you uploaded to your HTTP server during cluster
+installation.
+* Access to the PXE booting infrastructure that you used to create the machines
+for your {product-title} cluster during installation. The machines must boot
+from their local disks after {op-system} is installed on them.
+* If you use UEFI, you must have access to the `grub.conf` file that you modified
+during {product-title} installation.
+
+.Procedure
+
+. Confirm that your PXE or iPXE installation for the {op-system} images is
+correct.
+
+** For PXE:
++
+----
+DEFAULT pxeboot
+TIMEOUT 20
+PROMPT 0
+LABEL pxeboot
+    KERNEL http://<HTTP_server>/rhcos-<version>-installer-live-kernel-<architecture> <1>
+    APPEND ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-metal.<architecture>.raw.gz coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img coreos.inst.ignition_url=http://<HTTP_server>/worker.ign <2> <3>
```

Comment:
This is not correct. if using live kernel and initramfs, coreos.inst.install_dev needs to point to the full path (/dev/sda) and coreos.inst.image_url is not required.

Also ip=dhcp, rd.neednet and coreos.inst should be removed, as they are deprecated.

Correct line on 4.6 is:

initrd=http://\<HTTP_server\>/rhcos-\<version\>-installer-live-initramfs.\<architecture\>.img console=ttyS0 console=tty0 coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://\<HTTP_server\>/worker.ign coreos.live.rootfs_url=http://\<HTTP_server\>/rhcos-\<version\>-installer-live-rootfs.\<architecture\>.img